### ISSUE TITLE
Evaluation about mcunet-320KB(Imagenet)

### ISSUE THREAD

**Issue by andrewwang0612** [mit-han-lab/mcunet#11]
Thanks for the great work. 
I run this line to evluate the performance if this model
**python eval_torch.py --net_id mcunet-320kB --dataset {imagenet/} --data-dir PATH/TO/DATA/val**
**But the accuracy just gets about 11%,**

And I use this github to preprate the Imagenet dataset
**https://gist.github.com/antoinebrl/7d00d5cb6c95ef194c737392ef7e476a**
The validation just like this setting,it split to 1000 folders and each folder have about 50 images
![image](https://user-images.githubusercontent.com/87011223/189579949-fea5383f-a65e-4955-8578-dfe3ff8205e4.png)

Could you tell me  the possible reason?
Or I use the wrong way to split the Imagenet on validation?

**Comment by tonylins** [comment]
Hi, could you change the argument to `--dataset imagenet` (actually I think it will raise an error with your command)?

**Comment by andrewwang0612** [comment]
Oh, I haven't type it clearly.
I used the argument to `--dataset imagenet`

**Comment by tonylins** [comment]
This is weird. Can you try evaluating the VWW dataset, which I have provided as a zip copy?

**Comment by andrewwang0612** [comment]
I run this command and provided by yours VWW dataset
![image](https://user-images.githubusercontent.com/87011223/190175448-db24ed43-5e93-45d3-a4fa-91284006c44c.png)

python eval_torch.py --net_id mcunet-320kB --dataset vww --data-dir ./vww-s256/val --batch-size 4
![image](https://user-images.githubusercontent.com/87011223/190174993-ee25e538-5d77-4d8b-9281-db2866941a6c.png)
Still have some problem,I haven't modify eval_torch.py .

**Comment by tonylins** [comment]
Hi, for vww dataset, you need to test on vww models like `mcunet-320kB-vww`, instead of ImageNet models `mcunet-320kB`. Please use the following script instead:

`python eval_torch.py --net_id mcunet-320kB-vww --dataset vww --data-dir ./vww-s256/val --batch-size 4`

**Comment by andrewwang0612** [comment]
Thank you!!For vww dataset, I get the right accyracy.
But for imagenet dataset,still have the same problem!
Or I have to split 10,000 samples from the training set of ImageNet?

**Comment by tonylins** [comment]
Hi, thanks for confirming. I just pulled the repo and verified that I could reproduce the number. It should be related to the dataset processing. The number of iterations for testing also does not match. 
Can you try testing other `torchvision` models to see if you can get the correct results? Please see https://github.com/pytorch/examples/tree/main/imagenet and use `--evaluate`.
<img width="1295" alt="image" src="https://user-images.githubusercontent.com/14002639/190683534-86ed3d60-19c2-4219-9698-3a551073093f.png">

**Comment by andrewwang0612** [comment]
Thank you!! For imagenet dataset that provided by you.
I get the correct results.
